client/src/services/participant-details.js

```diff
@@ -25,6 +25,17 @@ const getInterestLabel = (isInterested) => {
   }
 };
 
+/**
+ * Function to make underscore-seperated strings title case with spaces.
+ * @param {string} status Status string from DB
+ * @returns Title case formatted status
+ */
+const getStatusLabel = (status) =>
+  status
+    .split('_')
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join(' ');
+
 export const getParticipantPageLabel = (pageName) => {
```

Comment:
Could you move this to a util file? Seems like it could be used elsewhere in the future.

Reply:
Potentially, yeah.

Reply:
(I assume you mean `getStatusLabel` and not `getParticipantPageLabel`.)

Reply:
Yes I did!